client/src/helpers/APICalls/request.ts

```diff
@@ -18,4 +18,21 @@ const fetchRequests = async (): Promise<RequestData> => {
     }));
 };
 
-export default fetchRequests;
+export const updateRequest = async (bookingStatus: string, requestId: string): Promise<RequestData> => {
+  const accepted = bookingStatus == 'accept' ? true : false;
+  const fetchOptions: FetchOptions = {
+    method: 'PATCH',
+    body: `{"accepted":${accepted},"_id": "${requestId}"}`,
+    headers: { 'Content-Type': 'application/json' },
+    credentials: 'include',
+  };
+  return await fetch(`/request`, fetchOptions)
```

Comment:
Aren't we sending the request id in the path parameters? :)

Reply:
Yes. Added it in.